diff --git a/node_modules/@payloadcms/storage-vercel-blob/dist/handleUpload.js b/node_modules/@payloadcms/storage-vercel-blob/dist/handleUpload.js
index d4d8c95..ded3818 100644
--- a/node_modules/@payloadcms/storage-vercel-blob/dist/handleUpload.js
+++ b/node_modules/@payloadcms/storage-vercel-blob/dist/handleUpload.js
@@ -1,9 +1,9 @@
-import { put } from '@vercel/blob';
+import { upload } from '@vercel/blob/client';
 import path from 'path';
 export const getHandleUpload = ({ access = 'public', addRandomSuffix, baseUrl, cacheControlMaxAge, prefix = '', token })=>{
     return async ({ data, file: { buffer, filename, mimeType } })=>{
         const fileKey = path.posix.join(data.prefix || prefix, filename);
-        const result = await put(fileKey, buffer, {
+        const result = await upload(fileKey, buffer, {
             access,
             addRandomSuffix,
             cacheControlMaxAge,
